src/utils/imageLargeView.html

```diff
@@ -0,0 +1,6 @@
+<div class="c-notebook-snapshot">
```

Comment:
Should we have some large image specific CSS?

Reply:
Ahh yes good catch, copy-paste error 👍